packages/widget/src/components/transfer/fungible/transfer-button/transfer-button.ts

```diff
@@ -7,13 +7,17 @@ import type { Button } from '../../../common';
 import { BaseComponent } from '../../../common';
 
 const enabledStates = [
-  FungibleTransferState.WRONG_CHAIN,
-  FungibleTransferState.WALLET_NOT_CONNECTED,
   FungibleTransferState.PENDING_APPROVALS,
   FungibleTransferState.PENDING_TRANSFER,
   FungibleTransferState.COMPLETED
 ];
 
+const disabledState = [
```

Comment:
wrong_chain and WALLET_NOT_CONNECTED shouldn't be here as in those cases click on the button will prompt wallet connection/chain switch.

I don't think you even need this var as any state that's not in enabled should be disabled state

Reply:
Now I got the logic, that we care only about "enabled" statuses, and all others are treated as errors 👍